<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ebrima</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ebrima</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GameOver</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>551</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Ebrima</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Continue?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Ebrima</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exit.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
